lib/commentOnClosedIssue.js

```diff
@@ -0,0 +1,62 @@
+'use strict';
+var Cesium = require('cesium');
+var Promise = require('bluebird');
+var rp = require('request-promise');
+
+var getUniqueMatch = require('./getUniqueMatch');
+
+var defined = Cesium.defined;
+
+/** Get comments -> regex search -> post comment
+ *
+ * @param {String} jsonResponse GitHub Response
+ * @param {Object} headers Request headers
+ * @return {Promise<http.IncomingMessage>} Response
+ */
+var commentOnClosedIssue = function(jsonResponse, headers) {
```

Comment:
You almost always want to use the `function xxx()` style.  The only time you may end up using the var style is when you are declaring a local function inside of another function.